packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md

```diff
@@ -6,6 +6,59 @@ title: Okta Identity Engine API Products release notes 2023
 
 > Help us improve our release notes by filling out this short [survey](https://surveys.okta.com/jfe/form/SV_4VEZcIGOX0TBgkC).
 
+## December
+
+### Monthly release 2023.12.0
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Devices API: new expand query parameter option](#devices-api-new-expand-query-parameter-option) | December 6, 2023 |
+| [Org detail retrieval and user app listing for My Account API is GA in Production](#org-detail-retrieval-and-user-app-listing-for-my-account-api-is-ga-in-production) | November 8, 2023 |
+| [MyAccount Authenticators API is GA in Preview](#myaccount-authenticators-api-is-ga-in-preview) | December 6, 2023 |
+| [Demonstrating Proof-of-Possession is GA in Production](#demonstrating-proof-of-possession-is-ga-in-production) | March 15, 2023 |
+| [Clarified RADIUS prompts](#clarified-radius-prompts) | December 6, 2023 |
+| [New possession constraint available for Policy API](#new-possession-constraint-available-for-policy-api) | December 6, 2023 |
+| [Bugs fixed in 2023.12.0](#bugs-fixed-in-2023-12-0) | December 6, 2023 |
+
+#### Devices API: new expand query parameter option
+
+The Devices API now includes a `userSummary` option for the `expand` query parameter in the [List all Devices](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Device/#tag/Device/operation/listDevices!in=query&path=expand&t=request) endpoint (`GET /api/v1/devices`). This new option returns a summary list of associated users for the device in the `_embedded` property. Previously, only the `user` option was available, which returned a full-detailed list of associated users. This new option allows API users to receive a smaller subset of key information for each user, thus improving performance without compromising functionality.
+<!-- OKTA-666753 -->
+
+#### Org detail retrieval and user app listing for My Account API is GA in Production
+
+Two new endpoints for the My Account API (`/idp/myaccount/organization` and `/idp/myaccount/okta-applications`) allow customers to retrieve org details and get a full list of the current user's Okta apps for a particular end user. See [My Account API](https://developer.okta.com/docs/api/openapi/okta-myaccount/guides/overview/). <!-- OKTA-651405, OKTA-663355, OKTA- -->
```

Comment:
```suggestion
Two new endpoints for the My Account API (`/idp/myaccount/organization` and `/idp/myaccount/okta-applications`) allow customers to retrieve org details and get a full list of the current user's Okta apps. See [My Account API](https://developer.okta.com/docs/api/openapi/okta-myaccount/guides/overview/). <!-- OKTA-651405, OKTA-663355, OKTA- -->
```
Maybe we don't need to add "for a particular end user" when you already identified that it's for the current user?

Reply:
@brentschaus-okta ?